chalicelib/checks/helpers/wfr_utils.py

```diff
@@ -201,6 +205,20 @@
           'fruit-fly': 'dm6',
           'chicken': 'galGal5'}
 
+# color map states bed file
+states_file_type = {'SPIN_states_v1': {'color_mapper': '/files-reference/4DNFI27WSLAG/', 'num_states': 9}}
+
+
+# check for a specific tag in a states file
+def isthere_states_tag(a_file):
```

Comment:
`a_file` by name and the comment above implies a file type but looks like its actually a dictionary. I would change the comment and the argument name so it's less ambiguous.